Tests/TuistKitTests/Generator/WorkspaceGeneratorTests.swift

```diff
@@ -91,19 +111,45 @@ final class WorkspaceGeneratorTests: XCTestCase {
     }
 
     func test_generate_relativeToGroup() throws {
-        try fileHandler.touch(path.appending(component: "Workspace.swift"))
-
         // Given the workspace manifest file exists
         // When generating the workspace
         // I expect the file to be relative to the workspace
 
-        try subject.generate(path: path, graph: graph, options: GenerationOptions())
+        try fileHandler.touch(path.appending(component: "Workspace.swift"))
+        try fileHandler.touch(path.appending(component: "Project.swift"))
+
+        let testWorkspace = Workspace.test(name: "test", contents: [
+            .file(path: path.appending(component: "Workspace.swift")),
+            .file(path: path.appending(component: "Setup.swift")),
+            .project(path: path),
+        ])
+
+        try subject.generate(workspace: testWorkspace, path: path, graph: graph, options: GenerationOptions())
 
         let workspace = try readWorkspace(at: path)
```

Comment:
I'd rather read it using `xcodeproj` and define the assertions on the parsed `XCWorkspace` object.